python/paddle/tensor/creation.py

```diff
@@ -980,6 +980,59 @@ def to_tensor(
             return _to_tensor_static(data, dtype, stop_gradient)
 
 
+class MmapStorage(paddle.base.core.MmapStorage):
+    """
+    This class will use mmap to load a file.
+    Args:
```

Comment:
```suggestion
    This class will use mmap to load a file.

    Args:
```
得空一行，否则文档渲染不成功
![image](https://github.com/user-attachments/assets/dc0655cc-18e2-4d60-9a98-7b2e61cdc9c9)

Reply:
行吧